engine/deeplearning/src/main/java/org/opennms/alec/engine/deeplearning/TFModel.java

```diff
@@ -146,36 +148,36 @@ private void loadModelHyperParameters(String modelPath) {
             List<Map<String,Object>> inputFeatures = (List<Map<String,Object>>)result.get("input_features");
             for (Map<String,Object> inputFeature : inputFeatures) {
                 if ("type_a".equals(inputFeature.get("name"))) {
-                    type_a_vocab = (List<String>)inputFeature.get("vocab");
+                    typeAVocab = (List<String>)inputFeature.get("vocab");
                 } else if ("type_b".equals(inputFeature.get("name"))) {
-                    type_b_vocab = (List<String>)inputFeature.get("vocab");
+                    typeBVocab = (List<String>)inputFeature.get("vocab");
                 }
             }
         } catch (IOException e) {
-            throw new RuntimeException("Failed to load model hyper parameters.", e);
+            throw new DeepLearningException("Failed to load model hyper parameters.", e);
```

Comment:
I'd opt to avoid adding a new checked exception here. This add a bunch of boilerplate, and can now lead to an NPE in calls to createEngine.